src/Core/LeanCode.Components.Startup/LeanStartup.cs

```diff
@@ -22,6 +23,12 @@ protected LeanStartup(IConfiguration config)
         Logger.Information("Configuration loaded, starting app");
     }
 
+    protected LeanStartup(IConfiguration config, bool closeAndFlushLogger)
```

Comment:
Let's remove this ctor and use the ability to override the property in derived classes - will be easier to use (and will have proper defaults).